datasets/my/consumer_alert_list/my_consumer_alert_list.yaml

```diff
@@ -39,6 +39,22 @@ data:
   format: HTML
 ci_test: false
 
+lookups:
+  type.name:
+    lowercase: true
+    normalize: true
+    options:
+      - match: "OctaFX Collecting agents of OctaFX as per following: i. Gavann Entertainment Sdn. Bhd. (1486933P) ii. Pro Niaga Consultation (SA0385991W) iii. Dass City Holdings Sdn. Bhd. (1486888W) iv. Mus Tech Trading Sdn. Bhd. (1472611D) v. Al Mierul International Sdn. Bhd. (1487076A) vi. Wan Energy Solution (M) Sdn. Bhd. (1486752D) vii. Taqara Solutions Sdn. Bhd. (1486752D)"
+        values: 
+          - OctaFX
+          - Gavann Entertainment Sdn. Bhd.
+          - Pro Niaga Consultation
+          - Dass City Holdings Sdn. Bhd.
+          - Mus Tech Trading Sdn. Bhd.
+          - Al Mierul International Sdn. Bhd.
+          - Wan Energy Solution (M) Sdn. Bhd.
+          - Taqara Solutions Sdn. Bhd.
```

Comment:
I don't think making these alternative names is accurate here - let's rather do "if len(name) > followthemoney.types.name.max_len then entity.add("description", name)" and then the `value` here would just be `OctaFX`